config/brokers/mt-channel-broker/deployments/controller.yaml

```diff
@@ -23,11 +23,11 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: mt-broker-controller
+      eventing.knative.dev/brokerRole: controller
   template:
     metadata:
       labels:
-        app: mt-broker-controller
+        eventing.knative.dev/brokerRole: controller
         eventing.knative.dev/release: devel
```

Comment:
2 reasons why I believe we should adopt that scheme:

* ingress and filter use `eventing.knative.dev/brokerRole`, not `app`

https://github.com/knative/eventing/blob/af999217845dc50de712fb88afdd898f5ebd081e/config/brokers/mt-channel-broker/deployments/broker-ingress.yaml#L26-L30

* IMC components use a consistent naming already

https://github.com/knative/eventing/blob/af999217845dc50de712fb88afdd898f5ebd081e/config/channels/in-memory-channel/deployments/controller.yaml#L25-L28

https://github.com/knative/eventing/blob/af999217845dc50de712fb88afdd898f5ebd081e/config/channels/in-memory-channel/deployments/dispatcher.yaml#L25-L28